lib/sudo-exec/src/lib.rs

```diff
@@ -30,10 +30,15 @@ pub fn run_command(ctx: Context<'_>, env: Environment) -> io::Result<ExitStatus>
     let mut command = Command::new(ctx.command.command);
     // reset env and set filtered environment
     command.args(ctx.command.arguments).env_clear().envs(env);
+    if let Some(path) = ctx.chdir {
+        // change current directory, if requested
+        command.current_dir(path);
+    } else if ctx.login {
+        // change current directory, if `-i` is being used.
+        command.current_dir(&ctx.target_user.home);
```

Comment:
this needs a `Path::exists` check because the home directory may not exist. in that case, ogsudo prints a warning (see failing test in CI) and still executes the command

Reply:
Yeah I noticed the failing CI. I already fixed it.  I noticed that ogsudo sets the current directory to `/` if the home directory does not exists so I also did that and updated the test to check that we set the directory to `/` and that the warning message is being printed.